components/tag-list/test/tag-list.vdiff.js

```diff
@@ -120,8 +116,10 @@ describe('tag-list', () => {
 		});
 
 		it('delete first item', async() => {
-			await sendKeys('press', 'Tab');
-			await sendKeys('press', 'Delete');
+			elem.focus();
+			sendKeys('press', 'Delete');
```

Comment:
Is the `await` no longer needed for `sendKeys`? Otherwise LGTM!

Reply:
I think the idea is that you don't need to `await` it if you're expecting it to trigger another event that you can wait for. Otherwise there's the possibility that the event you're waiting for could fire while you're still waiting for `sendKeys` to return.